tests/test_transpile.py

```diff
@@ -249,33 +250,24 @@ def test_index_offset(self, mock_logger):
         mock_logger.warning.assert_any_call("Applying array index offset (%s)", -1)
 
     def test_identity(self):
-        with open(
-            os.path.join(self.fixtures_dir, "identity.sql"), encoding="utf-8"
-        ) as f:
-            for sql in f:
+        self.assertEqual(transpile("")[0], "")
```

Comment:
ah i see you replaced my empty test

Reply:
Yeah - I made the fixture loader remove empty lines.
Felt quick and easy to just add this test in.